modules/api/src/main/java/org/eclipse/xpanse/api/config/GetCspInfoFromRequest.java

```diff
@@ -244,8 +242,8 @@ public Csp getCsFromWorkflowTaskId(String taskId) {
      */
     public Csp getCspFromManagementTaskId(String managementTaskId) {
```

Comment:
Delete this method and use `getCspFromServiceOrderId(String orderId)`.

Reply:
deleted.